src/executables/sysrepo-plugind.c

```diff
@@ -495,13 +495,13 @@ main(int argc, char **argv)
             if (get_plugins_dir(&plugins_dir)) {
                 goto cleanup;
             }
-            if (asprintf(&cmd, "/bin/cp -- \"%s\" %s", optarg, plugins_dir) == -1) {
+            if (asprintf(&cmd_arg, "-- \"%s\"", optarg) == -1) {
```

Comment:
`--` should probably be a separate argument.

Reply:
Fixed